README.md

```diff
@@ -98,6 +98,6 @@ Build the project and documentation:
 
     $ bundle exec jekyll build
 
-Or serve it on a local server on http://localhost:7998/Font-Awesome/:
+Or serve it on a local server on >http://localhost:7998/Font-Awesome/>:
```

Comment:
is `>http://localhost:7998/Font-Awesome/>` right?

Maybe it should be `<http://localhost:7998/Font-Awesome/>`

but we really don't want a link to localhost, maybe `` `http://localhost:7998/Font-Awesome` `` is better